### PR TITLE
Fix WAI-ARIA attributes in Combobox

### PR DESCRIPTION
**Pull request by sebpettersson**
Potential fix for #467

### CODE REVIEW

File: packages/combobox/src/index.tsx [hunk 1]
```diff
@@ -869,7 +870,6 @@ export const ComboboxButton = forwardRefWithAs<{}, "button">(
 
     return (
       <Comp
-        aria-controls={listboxId}
```
Comment (chancestrickland) [inline packages/combobox/src/index.tsx]: I believe we should leave this, even though it isn't in the ARIA example. We can probably make some other improvements to the button specifically for screen readers, but so long as it's here and controls the popover we should keep the aria context.

Comment (sebpettersson) [reply packages/combobox/src/index.tsx]: Sorry about that, I actually misread the code, it's fixed now.

Though now I'm unsure how ComboboxButton is used - it's not mentioned in the docs?

Comment (chancestrickland) [reply packages/combobox/src/index.tsx]: There's an example in the repo that shows how it's used, though I do need to think through its implementation a bit before we add documentation for it. The challenge AFAICT is that its functionality changes a bit depending on whether or not there is anything to render inside the popover, and that depends on how/whether or not your app decides to render a popover when matching options are not found. I'll have to chat with Ryan about some of these details to get some clarity before documenting that component and making it "official".

Comment (sebpettersson) [reply packages/combobox/src/index.tsx]: Got it, thanks for the explanation.

**Pull request merged**